__tests__/app.test.js

```diff
@@ -280,6 +319,36 @@ describe.only('PATCH /api/articles/:article_id', () => {
   })
 });
 
+describe('DELETE /api/comments/:comment_id', () => {
+  test('Returns 204 and empty object', () => {
+    const comment_id = 1;
+    return request(app)
+    .delete(`/api/comments/${comment_id}`)
+    .expect(204)
+    .then(({body}) => {
+      expect(body).toEqual({})
+    })
+  });
```

Comment:
A 204 means 'No content' and express will never send a response body back for this, so you don't have to make any assertions about the body